### ISSUE TITLE
Request for Training Hyperparameters to Reproduce CT-RATE Results

### ISSUE THREAD

**Issue by ethanbar11** [alibaba-damo-academy/fvlm#6]
# Request for Training Hyperparameters to Reproduce CT-RATE Results

### Details:

I'm working on implementing the "Fine-grained Vision-language Pre-training for Enhanced CT Image Understanding" model using the provided GitHub repository. I've successfully set up the data processing pipeline and prepared the CT-RATE dataset according to the instructions in the README.

I tried to reproduce results with the configuration in "lavis/projects/blip/train/pretrain_ct.yaml" but currently am unable to reproduce them (the loss doesn't go down).
To ensure exact reproduction of the reported results, I would like to kindly ask the exact config you guys used in order to produce the results.

Moreover, could you please clarify how currently the batches are distributed across GPUs and specifically if batch merging across multiple GPUs was implemented, as the current code doesn't seem to explicitly support this

#### Urgency:
High - I already ran a training run and was unable to reproduce the results and am planning to start training asap.

#### Additional context:
I've already downloaded and processed the CT-RATE dataset, the ViT weights, and the BiomedVLP-CXR-BERT-specialized text encoder. I've also prepared the directory structure as specified in the README.
Thank you for your assistance.

Ethan Baron
ethanbarbar [at] gmail.com

**Comment by windygoo** [comment]
Yes, the configuration file you mentioned ('lavis/projects/blip/train/pretrain_ct.yaml') is indeed the config we used for model training, and the training log is available [here](https://drive.google.com/file/d/1Ax0aaaejGFN2hfTqrDi3ISg19s_0Hbtr/view?usp=drive_link).

Have you attempted to reproduce the evaluation results using our provided model checkpoint? This can be used to verify if your data processing is correct.

**Comment by liweiyangv** [comment]
> # Request for Training Hyperparameters to Reproduce CT-RATE Results
> ## Subject: Request for exact hyperparameters to reproduce results on CT-RATE dataset
> ### Details:
> I'm working on implementing the "Fine-grained Vision-language Pre-training for Enhanced CT Image Understanding" model using the provided GitHub repository. I've successfully set up the data processing pipeline and prepared the CT-RATE dataset according to the instructions in the README.
> 
> I tried to reproduce results with the configuration in "lavis/projects/blip/train/pretrain_ct.yaml" but currently am unable to reproduce them (the loss doesn't go down). To ensure exact reproduction of the reported results, I would like to kindly ask the exact config you guys used in order to produce the results.
> 
> Moreover, could you please clarify how currently the batches are distributed across GPUs and specifically if batch merging across multiple GPUs was implemented, as the current code doesn't seem to explicitly support this
> 
> #### Urgency:
> High - I already ran a training run and was unable to reproduce the results and am planning to start training asap.
> 
> #### Additional context:
> I've already downloaded and processed the CT-RATE dataset, the ViT weights, and the BiomedVLP-CXR-BERT-specialized text encoder. I've also prepared the directory structure as specified in the README. Thank you for your assistance.
> 
> Ethan Baron ethanbarbar [at] gmail.com

Hi, I have also met the training problem, which is that the loss does not go down. Do you find a solution?
